workflows/argo/scream-prognostic-run.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: scream-prognostic-run
+spec:
+  entrypoint: driver
+  volumes:
+  - name: gcp-credentials-user-gcp-sa
+    secret:
+      secretName: gcp-key
+  - name: data-volume
+    persistentVolumeClaim:
+      claimName: scream-storage-claim
+  - name: dshm
+    emptyDir:
+      medium: Memory
+  templates:
+  - name: driver
+    inputs:
+      parameters:
+      - {name: config, value: "/tmp/180.yaml"}
```

Comment:
Does this default config file already exist on a volume that can be read when this job runs? 

Not sure if your usage is different and you don't have this issue, but one annoying difficulty for other argo templates was that it is not straightforward to pass a config file to the cluster for use in a job.
In other templates, to get around this we pass the entirety of the config's contents as the parameter (ex. [here](https://github.com/ai2cm/vcm-workflow-control/blob/1a27d5966ede468955fdc6e387eacdcadcd129dd/examples/train-evaluate-prognostic-run/run.sh#L19)).

Reply:
Yes, this is a file created inside docker as part of the pre-built executables. 

And yes, passing the entirety of the config's contents as the parameter is allowed. `scream_run prepare-config` takes care of that.